# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/xf86-video-nouveau/hotfix-sarea.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

From 3ee7cbca8f9144a3bb5be7f71ce70558f548d268 Mon Sep 17 00:00:00 2001
From: Ilia Mirkin <imirkin@alum.mit.edu>
Date: Mon, 26 Apr 2021 19:38:57 -0400
Subject: remove sarea.h usage

This header hasn't been used in ages, and is apparently no longer
shipped in some places. Just stop relying on it entirely.

Reported-by: Lyude Paul <lyude@redhat.com>
Signed-off-by: Ilia Mirkin <imirkin@alum.mit.edu>
---
 src/nv_include.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/nv_include.h b/src/nv_include.h
index de55570..eee8c5d 100644
--- a/src/nv_include.h
+++ b/src/nv_include.h
@@ -69,6 +69,5 @@
 #include "nv_type.h"
 #include "nv_proto.h"
 #include "nv_dma.h"
-#include "sarea.h"
 
 #endif /* __NV_INCLUDE_H__ */
-- 
cgit v1.2.1

